### PR TITLE
add daily cronjob to clean out job management

### PR DESCRIPTION
**Pull request by tomkralidis**
As a result of https://github.com/geopython/demo.pygeoapi.io/commit/49a426bb223264b290a1b28041c581b49a54c547 for GISE pgeoapi workshop preparation, adding cronjob to remove job management data on a daily basis.

Note: I have not done this for `services/pygeoapi_master/Dockerfile.alpine`, so any help would be appreciated for the alpine image (are we using it?  If not, should we delete it?).

### CODE REVIEW

**Review by doublebyte1 — APPROVED** [review]

File: services/pygeoapi_master/Dockerfile [hunk 1]
```diff
@@ -8,4 +8,13 @@ LABEL maintainer="Just van den Broecke <justb4@gmail.com>"
 COPY ./local.config.yml /pygeoapi/local.config.yml
 ADD ./data /pygeoapi/data
 # Should now be in pygeoapi Docker Image
+
```
Comment (justb4) [inline services/pygeoapi_master/Dockerfile:11]: IMO lines 11-19 can be made much simpler, only one line: 

`COPY ./demo.pygeoapi.io.cron /etc/cron.d/demo.pygeoapi.io.cron`.

*  `/etc/cron.d` is in container already, with jobs
* `crontab` program/command is not required when already copying to `/etc/cron.d/`
* file permissions are preserved in GH, no need for `chmod`

File: services/pygeoapi_master/demo.pygeoapi.io.cron [hunk 2]
```diff
@@ -0,0 +1 @@
+0 1 * * * /bin/rm -f /tmp/pygeoapi-process-manager.db > /proc/1/fd/1 2>/proc/1/fd/2
```
Comment (justb4) [inline services/pygeoapi_master/demo.pygeoapi.io.cron:1]: can be simpler: `/bin/rm -f /tmp/pygeoapi-process-manager.db > /dev/null 2>&1`

**Review by justb4 — CHANGES_REQUESTED** [review]
See my proposed changes in Dockerfile and cronfile.

`Dockerfile.alpine` is not used, we can even remove, as Alpine base images for Dockerfiles are not necessary anymore with slim Debian/Ubuntu base images.

File: services/pygeoapi_master/demo.pygeoapi.io.cron (hunk 2 above)
Comment (tomkralidis) [reply services/pygeoapi_master/demo.pygeoapi.io.cron]: Fixed.

**Comment by tomkralidis** [comment]
Thanks @justb4.  Updates pushed, for review.

**Review by justb4 — APPROVED** [review]
Proof will be in the pudding, on demo server can check:

* `docker exec -it pygeoapi_master ls -l /etc/cron.d`
* `docker exec -it pygeoapi_master ls -l --time=birth /tmp/pygeoapi-process-manager.db`

**Comment by justb4** [comment]
I can check as this Docker image is only rebuilt/deployed when the pygeoapi image is renewed, we still need GH Workflows, see issue #36.

**Comment by justb4** [comment]
Hmm, I think we/I overlooked that `cron` is not available within Ubuntu on Docker. I think even if we install `cron` in the Dockerfile, the jobs (`cron` daemon) won't run. Remember how in the first versions of GeoHealthCheck we had a special utility/Docker base for this? 

There are many references how to do this, but quite involved as a Docker container is supposed to run a single process. I also worked on several solutions. One is using [supervisor](https://gdevillele.github.io/engine/admin/using_supervisord/) for example an `AWStats` Docker Image with scheduled stats generation and serving these with Apache: https://github.com/justb4/docker-awstats . In a conf one can specify processes to run:
https://github.com/justb4/docker-awstats/blob/master/confs/supervisord.conf
Maybe there are other solutions these days for cron in Docker, or for scheduled jobs. 

Using the demo base OS cron with a `docker exec pygeoapi_master rm -f ...` is another possibility, but IMO not very clean.

**Pull request merged**